kahuna/public/js/edits/labeller.css

```diff
@@ -17,13 +19,18 @@
     background-color: #99ddff;
 }
 
-.label--partial,
-.label--partial gr-icon {
+/*
+label--partial is part of gr-panel.
+TODO refactor gr-panel to use common labeller component
+*/
+.label--partial .label__value,
```

Comment:
Could we not just apply this to `label--partial`? Why have every building block have it?